src/_includes/docs/install/compiler/android.md

```diff
@@ -176,7 +176,7 @@ location of your Java installation.
 Android sdkmanager tool was found, but failed to run
 (/Users/atsansone/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager): "exited code 1".
 Try re-installing or updating your Android SDK,
-visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions.
+visit https://docs.flutter.dev/get-started/install/macos/mobile-android#configure-android-development for detailed instructions.
```

Comment:
Can you update this to match the new link from the `flutter` tool? As this is an example of the output. Thanks!

```suggestion
visit https://flutter.dev/to/macos-android-setup for detailed instructions.
```

Reply:
Sure. Thank you